torizoncore-builder.py

```diff
@@ -30,6 +30,14 @@
 from tcbuilder.errors import TorizonCoreBuilderError, InvalidArgumentError
 # pylint: enable=wrong-import-position
 
+# if the TCB_DEBUGPY environment variable is set
+# means that a developer is trying to debug the code from outside
+# so, we wait for a debugger to attach to the process
+if os.getenv("TCB_DEBUGPY"):
+    import debugpy
+    debugpy.listen(("0.0.0.0", 5678))
+    debugpy.wait_for_client()
+
```

Comment:
Developers. This should not have any impact if the `TCB_DEBUGPY` is not set, and adds a great value. I'm not a TCB expert and I was able to make this contribution in "record" time, let's say, only because I was able to attach to debug and follow the flow to better understand what was happening.

Reply:
Ok, seems sensible to me.